File: richest-customer-wealth/richest-customer-wealth.go
func maximumWealth(accounts [][]int) int {
  maxWealth := 0

  for _, customer := range accounts {
    totalMoney := 0
    for _, money := range customer {
      totalMoney += money
    }

    if maxWealth < totalMoney {
      maxWealth = totalMoney
    }
  }

  return maxWealth
}

File: richest-customer-wealth/richest-customer-wealth.cpp
class Solution {
public:
  int maximumWealth(vector<vector<int>>& accounts) {
    int maxWealth = 0;

    for ( auto customer : accounts ) {
      int countMoney = 0;
      for ( auto money : customer ) {
        countMoney += money;
      }

      maxWealth = maxWealth < countMoney ? countMoney : maxWealth;
    }

    return maxWealth;
  }
};